README.md

```diff
@@ -10,6 +10,64 @@ under [Project Quarks].
 [Cloud Foundry Operator]: https://github.com/cloudfoundry-incubator/cf-operator/
 [Project Quarks]:         https://www.cloudfoundry.org/project-quarks/
 
+## Contributing to Kubecf development
+
+See the [Guide to Contribution](doc/Contribute.md).
+
+# System requirements
+
+To work with `kubecf` a variety of supporting tools are required. These are
+
+| Tool		| Notes					|
+|---		|---					|
+|k8s		| Foundation.				|
+|minikube	| Provider for local k8s clusters.	|
+|kind		| Provider for local k8s clusters.	|
+|kubectl	| Client to talk to k8s clusters.	|
+|Helm (client)	| Handling helm charts.			|
+|Helm (server)	| __Tiller is not required__.		|
```

Comment:
```suggestion
|Helm Tiller	| Necessary with Helm 2.x.		|
```

Reply:
I believe that this change would be in conflict with your

> I'm saying that Tiller is not needed for kubecf itself. With kubecf we use the helm template command and pipe the result to kubectl apply.

in the jira ticket.

Reply:
Might be better to say `Necessary with Helm 2.x. when using external charts`.

Reply:
OK, I'll try to be more clear. Tiller is the "deployer" of Helm 2.x. It is the server-side component that talks to the Kubernetes API. It's the standard way of deploying when using Helm 2.x.

The confusion is around "Helm using KubeCF" vs "KubeCF using Helm".

It's certainly Helm that uses the KubeCF chart to deploy it, not the other way around. When I say that `kubecf uses the helm template command`, I mean that the scripts under the kubecf repo use the `helm template` command to deploy KubeCF.

People familiar with the process of deploying a Helm chart using Helm 2.x will certainly have Tiller installed.

My bad on not being clear in the Jira ticket. Sorry.

Reply:
Hopefully addressed by commit d256a8546bac8d48a164f62afd1b065ce7cc7ebf